app/javascript/dashboard/assets/scss/widgets/_search-box.scss

```diff
@@ -1,18 +1 @@
-.search {
```

Comment:
We are not using this class name anywhere. I think it's related to our old search.